webapp/src/components/App/Singlestudy/explore/Modelization/BindingConstraints/BindingConstView/ConstraintFields.tsx

```diff
@@ -0,0 +1,161 @@
+import { Box } from "@mui/material";
+
+interface Props {
+  study: StudyMetadata;
+  constraintId: string;
+  isMatrixOpen: boolean;
+  onCloseMatrix: VoidFunction;
+}
+
+function Fields({ study, constraintId, isMatrixOpen, onCloseMatrix }: Props) {
+  const { t } = useTranslation();
+  const { control, getValues } = useFormContextPlus<BindingConstraint>();
+  const currentOperator = getValues("operator");
+
+  const outputFilterOptions = useMemo(
+    () =>
+      OUTPUT_FILTERS.map((filter) => ({
+        label: t(`global.time.${filter}`),
+        value: filter,
+      })),
+    [t],
+  );
+
+  const operatorOptions = useMemo(
+    () =>
+      OPERATORS.map((operator) => ({
+        label: t(`study.modelization.bindingConst.operator.${operator}`),
+        value: operator,
+      })),
+    [t],
+  );
+
+  const timeStepOptions = useMemo(
+    () =>
+      TIME_STEPS.map((timeStep) => ({
+        label: t(`global.time.${timeStep}`),
+        value: timeStep,
+      })),
+    [t],
+  );
+
+  ////////////////////////////////////////////////////////////////
+  // JSX
+  ////////////////////////////////////////////////////////////////
+
+  return (
+    <>
+      <Fieldset
+        fieldWidth={200}
+        sx={{ py: 1, display: "flex", flexWrap: "wrap" }}
+      >
+        <StringFE
+          disabled
+          name="name"
+          label={t("global.name")}
+          size="small"
+          control={control}
+          rules={{ validate: (v) => validateString(v) }}
+          sx={{ m: 0 }} // TODO: Remove when updating MUI Theme
+        />
+        {Number(study.version) >= 870 && (
+          <StringFE
+            name="group"
+            label={t("global.group")}
+            size="small"
+            control={control}
+            rules={{
+              validate: (v) =>
+                validateString(v, {
+                  max: 20,
+                  specialChars: "-",
+                }),
+            }}
+            sx={{ m: 0 }} // TODO: Remove when updating MUI Theme
+          />
+        )}
+        <StringFE
+          name="comments"
+          label={t("study.modelization.bindingConst.comments")}
+          size="small"
+          control={control}
+          required={false}
+          sx={{ m: 0 }} // TODO: Remove when updating MUI Theme
+        />
+        <SelectFE
+          name="timeStep"
+          label={t("study.modelization.bindingConst.type")}
+          size="small"
+          variant="outlined"
+          options={timeStepOptions}
+          control={control}
+        />
+        <SelectFE
+          name="operator"
+          label={t("study.modelization.bindingConst.operator")}
+          size="small"
+          variant="outlined"
+          options={operatorOptions}
+          control={control}
+        />
+        <SwitchFE
+          name="enabled"
+          label={t("study.modelization.bindingConst.enabled")}
+          control={control}
+        />
+        {Number(study.version) >= 840 && (
```

Comment:
Fields `filter_year_by_year` and `filter_synthesis` are only available for study v8.3 or above, not v8.4

See File format changes [v8.3.2 — Marginal price of a binding constraint](https://antares-simulator.readthedocs.io/en/latest/reference-guide/13-file-format/#marginal-price-of-a-binding-constraint)

Reply:
@laurent-laporte-pro be sure it's good though. because I'm not sure the simulator versions match ours exactly. And it was legacy code that had been there for a while, so if there was a problem it would have been brought to our attention, I guess.